style.css

```diff
@@ -0,0 +1,312 @@
+* {
+	box-sizing: border-box;
+}
+
+
+body {
+  margin: 0;
+  background: #aabbfb;
+  font-family: "Source Sans Pro";
+  font-size: 16px;
+}
+
+.nav-drop {
+	display: none;
+}
+
+.dropbtn {
+	background-color: #aae4fb;
+	color:  black;
+	padding: 6.5px;
+	font-size: 1.8rem;
+	border: 0.5px solid;
+	padding-right: 20px;
+	padding-left: 20px;
+	padding-top: 8px;
+	margin-bottom: 5px;
+	border-radius: 8px;
+	margin-left: 5px;
+}
+
+.dropdown-content {
+	display: none;
+	position: absolute;
+	background-color: #aae4fb;
+	width: 165px;
+	box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+	z-index: 2;
+	margin-top: -5px;
+	margin-left: 5px;
+	border-radius: 8px;
+}
+
+.dropdown-content a {
+	color: black;
+	padding: 12px 16px;
+	text-decoration: none;
+	display: block;
+}
+
+.dropdown-content a:hover {
+	background-color:  #c2aafb
+}
+
+.nav-drop:hover .dropdown-content {
+	display: block
+}
+
+.nav-drop:hover .dropbtn {
+	background-color: #c2aafb;
+}
+
+.nav-container {
+	display: flex;
+	background-color: #6181f8;
+	justify-content: space-between;
+	height: 50px;
+	text-align: center;
+	width: 100%;
+}
+
+.nav-links {
+	display: flex;
+	justify-content: space-between;
+	text-align: center;
+	margin-left: 10px;
+}
+
+.nav-links a {
+	text-decoration: none;
+	color: black;
+	padding: 15px;
+	font-size: 18px;
+	font-family: Helvetica;
+	padding-top: 16px;
+	transition: transform .2s;	
+}
+
+.nav-links a:hover {
+	transform: scale(1.2);
+}
+
+.nav-cv {
+	border: 3px solid black;
+	border-radius: 8px;
+	margin-right: 10px;
+	width: 130px;
+	height: 40px;
+	padding-top: 10px;
+	margin-top: 5px;
+}
+
+.nav-cv a {
+	text-decoration: none;
+	color: black;
+	font-size: 1.1rem;
+	font-family: Helvetica;
+	margin-top: 10px;
+}
+
+.nav-cv a:hover {
+	color: #ffffff;
+}
+
+header {
+	display: flex;
+	width: 100%;
+	height: 800px;
+	background-image: url("coffee.jpg");
+	background-repeat: no-repeat;
+	background-size: cover;
+	background-position: center;
+	margin-top: 0px;
+	margin-bottom: 20px;
+	text-align: center;
+	align-items: center;
+	justify-content: center;
+}
+
+h1 {
+	font-family: "Josefin Sans";
+	font-size: 4rem;
+	color: #3b2a8b;
+}
+
+.about-container {
+	display: flex;
+	justify-content: center;
+	flex-direction: row;
+	height: 600px;
+}
+
+.about-writing {
+	display: flex;
+	width: 50%;
+	text-align: center;
+	justify-content: center;
+	flex-direction: column;
+	align-items: center;
+}
+.about-writing h3 {
+	font-family: "Montserrat";
+	font-size: 2rem;
+}
+
+.about-writing p {
```

Comment:
For the typography within the website I would recommend within the css sheet making a seperate p, h1, h2 h3 (and so on) section, with all the font details in. So later on when you use the p tag or the h2 tag the font is applied straight away. 
Here you have used ".about-writing p". if you did a generic "p" at the top of your css it means you won't have to add similar information throughout the css page.